services/masterdata/src/lib/core/services/masterdata-crud-http.service.ts

```diff
@@ -85,6 +86,17 @@ export class MasterdataCrudHttpService {
 			catchError(this.handleError<Masterdata>("getMasterdataById", masterdataModelAdapter.adapt(null)))
 		);
 	}
+
+	getAllTags(masterdatatype: string, offset: number, rows: number): Observable<TagsResultList> {
+		const tagsResultListModelAdapter = new TagsResultListModelAdapter();
+
+		const url = this.genericListUriMasterdata(`${this.baseUrl}${!isEmpty(masterdatatype) ? "/" + masterdatatype : ""}/tags`, offset, rows);
+		return this.client.get<TagsResultList>(url).pipe(
+			tap(() => this.log(null, "getAllTags", "success")),
```

Comment:
I don't think we need to log this everytime. (Un)successful networkrequest can be viewed in the browser devtoolbar

Reply:
i will let this for now. i wil remove this in a future story!

Reply:
Which future story?

Reply:
the one which will be for "adding translation" to this masterdata service.